static/styles/zulip.css

```diff
@@ -3324,3 +3324,12 @@ textarea.invitee_emails {
         }
     }
 }
+
+.space-between {
+    display: flex !important;
+    justify-content: space-between;
+}
+
+.space-between strong {
+    margin-right: 14px;
+}
```

Comment:
This CSS could result in unwanted results, so let's limit defining CSS for topic typeahead inside the CSS class for it.

Reply:
So rather than styling them with a element selector, I should give the element a class and styling them with it. Is that correct?

Reply:
It is fine to style the topic typeahead with `space-between` but you should limit the scope of the css to `.typeahead-menu`.

Reply:
Ahh that makes sense. I've made the final changes. Should it be good for integration review?

Reply:
I pushed a change to implement what I was thinking, please take a look!